src/pages/messages/components/TicketTypeTag.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box } from '@mui/material';
+import { TicketType } from '../../../types/publication_types/ticket.types';
+import { getTicketColor, getTicketTypeIcon } from '../utils';
+import { useTicketTypeText } from '../../../utils/hooks/useTicketTypeText';
+
+interface TicketTypeTagProps {
+  type: TicketType;
+  noText?: boolean;
```

Comment:
This is so we can only add a prop if the tag is to be used without text, which is a less common use case.